CesiumGltfReader/src/Reader.cpp

```diff
@@ -352,10 +353,13 @@ CesiumGltf::readImage(const gsl::span<const std::byte>& data) {
       &channelsInFile,
       image.channels);
   if (pImage) {
-    image.pixelData.assign(
+    const int lastByte =
```

Comment:
Can you add an explanation or small comment for this code change? I see current code clearer and more concise than the proposed change. It's a personal opinion though.

Reply:
`std::transform` is necessary here because we decided to change all of our uses of `std::uint8_t` to `std::byte`. `std::uint8_t` is not implicitly convertible to `std::byte` (intentional choice by the cpp powers that be). Transform allows us to use a lambda to perform the explicit conversion here. I added a clarifying comment.